.zenodo.json

```diff
@@ -92,6 +95,9 @@
       "name": "Goncalves, Mathias",
       "orcid": "0000-0002-7252-7771"
     },
+    {
+      "name": "Riddell, Cameron"
```

Comment:
@CRiddler Please let me know if you would like your name presented differently, and if you have an affiliation/ORCID that you want associated with your contributions.

This will be how your name is presented as an author on the [Zenodo archive](https://doi.org/10.5281/zenodo.591597). If you prefer not to be an author, let me know.

Reply:
You can update mine to this:

```
    {
      "affiliation": "Department of Psychology, University of California Davis, CA, USA",
      "name": "Riddell, Cameron",
      "orcid": "0000-0001-8950-0375"
    }
```

Thanks!